azure-functions/acbs-function/src/functions/acbs-issue-facility.js

```diff
@@ -85,20 +85,20 @@ df.app.orchestration('acbs-issue-facility', function* issueFacility(context) {
             const input = acbsFacilityFeeInput[i];
             facilityFee.push(
               yield context.df.callActivityWithRetry('create-facility-fee', retryOptions, {
-                facilityIdentifier: facilityId,
+                facilityIdentifier,
                 acbsFacilityFeeInput: input,
               }),
             );
           }
         } else {
           facilityFee = yield context.df.callActivityWithRetry('create-facility-fee', retryOptions, {
-            facilityIdentifier: facilityId,
+            facilityIdentifier,
             acbsFacilityFeeInput,
           });
         }
 
         return {
-          facilityId: facility._id,
+          facilityIdentifier: facility._id,
```

Comment:
Finding this a bit confusing without context, why is `facilityIdentifier` here not the same as other instances? Maybe could add a comment?

Reply:
Good spot, I missed it out have removed `facility._id`.